### PR TITLE
Fixes for auth

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/Membership/MembershipRequest.php [hunk 1]
```diff
@@ -106,9 +105,6 @@ class MembershipRequest implements MembershipInterface
 
     public function __construct()
     {
-        $this->gender = Genders::MALE;
```
Comment (ottaviano) [inline src/Membership/MembershipRequest.php]: maybe assign the default values on property?
`private $gender = Genders::MALE`?

File: src/Membership/MembershipRequest.php [hunk 2]
```diff
@@ -165,7 +161,7 @@ public function setEmailAddress(string $emailAddress): void
 
     public function getEmailAddress(): string
     {
-        return $this->emailAddress;
+        return $this->emailAddress ?: '';
```
Comment (ottaviano) [inline src/Membership/MembershipRequest.php]: `?? '';`

Comment (B-Galati) [reply src/Membership/MembershipRequest.php]: Why, in this case it's exactly the same thing no ?

File: src/Membership/MembershipRequest.php (hunk 1 above)
Comment (B-Galati) [reply src/Membership/MembershipRequest.php]: We don't want a default value anymore in this case.

File: src/Membership/MembershipRequest.php (hunk 2 above)
Comment (ottaviano) [reply src/Membership/MembershipRequest.php]: yes, in this case it do the same job, just for using php7

**Pull request merged**